riscv-sbi.adoc

```diff
@@ -257,6 +290,291 @@ call doesn't return.
 | *RESERVED*                |             |    0x09-0x0F |
 |===
 
+
+== Timer Extension, Extension ID: 0x54494D45 (TIME)
+This replaces legacy timer extension (0x00). It follows the new calling convention
+defined in v0.2.
+
+[source, C]
+----
+struct sbiret sbi_set_timer(uint64_t stime_value)
+----
+Programs the clock for next event after *stime_value* time. *stime_value* is in absolute
+time. This function must clear the pending timer interrupt bit as well.
```

Comment:
This needs some rewriting.  If you move your PR over to a branch in this repo then I'm happy to do so, but I don't think I can do that on one you own.

Reply:
Sure. But I don't have write access to the repo so I can't create a branch in this repo.
Can you grant me write access or create a branch so that I can push it to that branch ? I am fine with the update in here as well.